server/index.js

```diff
@@ -118,7 +128,7 @@ module.exports = config => {
 
     // route to get server info
     app.get('/server', (req, res) =>
-      res.status(200).send({ bcoinUri: config.uri })
+      res.status(200).send({ bcoinUri: bcoinConfig.str('node-uri') })
```

Comment:
Maybe we should have the fallback be defined in this case of something like `not set`?